addons/iot_box_image/overwrite_before_init/etc/init_image.sh

```diff
@@ -84,6 +85,33 @@ pip() {
   fi
   pip3 \"\$1\" \"\$2\" --break-system-package \$additional_arg
 }
+
+devtools() {
+  if [[ -z \"\$1\" || -z \"\$2\" || \"\$1\" == \"-h\" ]]; then
+    echo 'Usage: devtools <enable/disable> <general/actions> [action name]'
+    echo ''
+    echo 'Only provide an action name if you want do enable/disable a specific device action.'
+    echo 'If no action name is provided, all actions will be enabled/disabled.'
+    return 1
+  fi
+  write_mode
+  if ! grep -q '^\[devtools\]' /home/pi/odoo.conf; then
+    printf '\n[devtools]\n' >> /home/pi/odoo.conf
+  fi
+  if [ \"\$1\" == \"disable\" ]; then
+    if [[ -z \"\$3\" ]]; then
+      value=\"*\"
+    else
+      value=\"\$3\"
+    fi
+    devtools enable \$2
```

Comment:
the `enable` within `disable` is confusing, we should explain that we overwrite the existing value here